src/commands/new.ts

```diff
@@ -1,140 +1,126 @@
 import { CommandExists, CreateFileFromTemplate } from '#functions';
 import { PromptNew } from '#prompts';
-import { Command, flags } from '@oclif/command';
-import { blueBright, red } from 'chalk';
+import chalk from 'chalk';
 import { exec, spawn } from 'child_process';
 import { existsSync } from 'fs';
 import { cp, readFile, rm, writeFile } from 'fs/promises';
 import ora from 'ora';
 import { resolve } from 'path';
 import prompts from 'prompts';
-import { config } from '../config';
-
-export default class New extends Command {
-	public async run() {
-		const { args, flags } = this.parse(New);
-		const response = await prompts(PromptNew(args.ProjectName, await CommandExists('yarn')));
-		if (!response.projectName || !response.projectLang || !response.projectTemplate || !response.packageManager) {
-			process.exit(1);
-		}
-		const projectName = response.projectName === '.' ? '' : response.projectName;
+import { repoUrl } from '#constants';
+
+const { blueBright, red } = chalk;
 
-		const stpJob = async () => {
-			await cp(`./${response.projectName}/ghr/examples/${response.projectTemplate}/.`, `./${response.projectName}/`, { recursive: true });
+function editPackageJson(location: string, name: string) {
+	return new Promise(async (resolve, reject) => {
+		const pjLocation = `./${location}/package.json`;
+		const output = JSON.parse(await readFile(pjLocation, 'utf8'));
+		if (!output) return reject(new Error("Can't read file."));
 
-			for (const p of ['.gitignore', '.prettierignore']) {
-				await cp(`./${response.projectName}/ghr/${p}`, `./${response.projectName}/${p}`, { recursive: true });
+		output.name = name;
+		await writeFile(pjLocation, JSON.stringify(output, null, 2)).catch(reject);
+		return resolve(true);
+	});
+}
+
+function installDeps(location: string, pm: string, verbose: boolean) {
+	const pmp = spawn(pm.toLowerCase(), ['install'], {
+		stdio: verbose ? 'inherit' : undefined,
+		cwd: `./${location}/`
+	});
+
+	return new Promise((resolve, reject) => {
+		pmp.on('error', reject);
+
+		pmp.on('exit', async (code) => {
+			if (code === 0) {
+				const lockfile = `./${location}/${pm === 'npm' ? 'yarn.lock' : 'package-lock.json'}`;
+				if (existsSync(lockfile)) await rm(lockfile);
```

Comment:
Uhm. *no*? Not only is this sync/async, why would you delete the lockfile???

Reply:
If you choose yarn you should not have package-lock.json and same for npm/yarn.lock

Reply:
OH, ok now I see what it was. It's deleting the reverse lock file. Can you at least use the async exists instead of the sync one? 🙏

Reply:
> OH, ok now I see what it was. It's deleting the reverse lock file. Can you at least use the async exists instead of the sync one? 🙏

fs.exists is deprecated

Reply:
Is there no async version of this?? Really? :C

Reply:
¯\_(ツ)_/¯

Reply:
There isn't @vladfrangu  :( I ran into that just recently myself as well.

Reply:
imo it would be better if it stayed like this